rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/internal_interpreter.kt

```diff
@@ -1266,6 +1274,11 @@ class InternalInterpreter(val systemInterface: SystemInterface) {
                     else -> throw UnsupportedOperationException("I do not know how to handle $value with %INT")
                 }
             }
+            is RemExpr -> {
+                val n = eval(expression.dividend).asInt().value
+                val m = eval(expression.divisor).asInt().value
+                return IntValue(n % m)
```

Comment:
The modulo operator in Java behaves in a peculiar way for negative values. Is that mapping to what the REM statement does in RPG?

Reply:
Excellent observation: I added a test, and it seems it works fine.